src/Microsoft.SqlTools.Authentication/Utility/Encryption/EncryptionUtils.cs

```diff
@@ -0,0 +1,76 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Security.Cryptography;
+
+namespace Microsoft.SqlTools.Authentication.Utility
+{
+    public static class EncryptionUtils
```

Comment:
I wonder if this utility code is better in the credential assembly in case it's needed somewhere else?  or does this assembly not reference that one?

Reply:
Yeah, this one doesn't reference any of the service assemblies, only a 'Microsoft.SqlTools.Shared' project for shared constants, logger and utils.